testsuite/features/secondary/min_salt_migration.feature

```diff
@@ -97,9 +97,15 @@ Feature: Migrate Salt to bundled Salt on a nested Minion VM
     And I expand the results for "salt_migration_minion"
     Then I should see "/etc/salt: directory" in the command output for "salt_migration_minion"
 
-  Scenario: Migrate the nested VM to the Salt bundle and check the result
+  Scenario: Migrate the nested VM to the Salt bundle
+    When the Salt master can reach "salt_migration_minion"
```

Comment:
```suggestion
    Then the Salt master can reach "salt_migration_minion"
```

Reply:
It still looks weird to me to start a scenario with `Then`. From a logical perspective it has to haben something before a `Then`.

Reply:
Sorry, not `Then` but `Given`. As this is your pre-condition.

Reply:
I see. That makes more sense.

Reply:
Fixed.